R/ds_upsert.R

```diff
@@ -0,0 +1,31 @@
+#' Upsert data to a datastore
+#'
+#' @export
+#' @param resource_id (string) Resource id that the data is going to be stored
+#'   against.
+#' @param force (logical) Set to \code{TRUE} to edit a read-only resource.
+#'   Default: FALSE
+#' @param method (string) Set to \code{insert}, \code{upsert} or \code{update}
```

Comment:
can you add either here or in `@details` or `@section` what these different options are for the `method` param